src/Module.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace LotGD\Core;
+
+/**
+ * Classes which provide module functionality should implement this interface.
+ */
+interface Module extends EventHandler
```

Comment:
A module is limited to exactly 1 event handler in the main class, which then distributes events by itself on different subhandler?